api_spec.yaml

```diff
@@ -2302,6 +2296,8 @@ paths:
                   $ref: '#/components/schemas/userDataInDatabase'
                 enableAntiCsrf:
                   $ref: '#/components/schemas/enableAntiCsrf'
+                useStaticSigningKey:
+                  $ref: '#/components/schemas/useStaticSigningKey'
```

Comment:
I think we should do that separately, but I can also add it here.

Reply:
How would it be done separately? Don't we need these values during session creation?

Reply:
They are set in the core configuration - making them controllable from the backend is a change.
We can do it, but we also need to decide a few things about them, for example, the validation/possible values. 
A discussion about this is in my backlog.

Reply:
Yea, we should discuss this. I was under the impression that this would be changed in this PR, and that the core config for these lifetimes would be removed (i remember you said this during our discussion on multi tenancy). So im confused as to why it's not here.

If you think this is a lot of work for now, we can do it later - but we should discuss this.

Reply:
We decided to do this later.